hoot-services/src/main/java/hoot/services/controllers/conflation/AdvancedConflationOptionsResource.java

```diff
@@ -101,6 +107,14 @@
         put("differential w/Tags", DIFFERENTIAL_CONFLATION_PATH);
     }};
 
+    private Map<String, String> matcherMap = new HashMap<String, String>(){{
+        put("GenericLines", "hoot::LinearCriterion");
+        put("PowerLines", "hoot::PowerLineCriterion");
+        put("Railways", "hoot::RailwayCriterion");
+        put("Rivers", "hoot::LinearWaterwayCriterion");
+        put("Roads", "hoot::HighwayCriterion");
+    }};
```

Comment:
I'm asking myself what the point of running `hoot.bin info --way-snap-criteria` in the command is if the output is also hardcoded here?

Reply:
honestly its a good point. Initially I was doing it with the command and as it went I needed to add mappings like this to make things work

Reply:
Could you just take `hoot:LinearWaterwayCriterion` and strip out `hoot:` and `Criterion` and then add spaces before capital letters to get your combobox label?

Reply:
Then it would be dynamic based on hoot command which is preferable

Reply:
So in the `AdvancedConflationOptionsResource.java` I have `displayToHootMap` and that is for populating the combobox label which is done the way you said except maybe i didnt take criterion on.

But the problem is that I need this `matcherMap` because the combobox is populated based on what matchers are selected so that's how I can know that if `Rivers` is not selected then the combobox should not contain `hoot::LinearWaterwayCriterion`

Reply:
Ah, I didn't realize it was that sophisticated :mage_man: